src/media-popup.js

```diff
@@ -1,21 +1,33 @@
 // FIXME: Refactor later
-var ipcRenderer = require('electron').ipcRenderer;
+import { ipcRenderer }        from 'electron';
```

Comment:
Will change again, thought i wanted to do it like you have in your other files in the top but nvm. will change again.